explorer/app/components/SocialLinks/icons.tsx

```diff
@@ -0,0 +1,74 @@
+export const Twitter = () => (
```

Comment:
Let's update these svg references (see my comments below for `explorer/images/discourse.svg`).